packages/preload/src/index.ts

```diff
@@ -829,6 +829,13 @@ function initExposure(): void {
     return ipcInvoke('command-registry:executeCommand', command, ...args);
   });
 
+  contextBridge.exposeInMainWorld(
+    'executeCommandWithFeedback',
+    async (command: string, ...args: unknown[]): Promise<void> => {
+      return ipcInvoke('command-registry:executeCommandWithFeedback', command, ...args);
+    },
+  );
```

Comment:
I'm not sure to fully understand what means 'withFeedback' 

I execute a command, that's all.